packages/desktop-client/src/hooks/useSelected.tsx

```diff
@@ -246,14 +246,14 @@ export function useSelected<T extends Item>(
 }
 
 const SelectedDispatch = createContext<(action: Actions) => void>(null);
-const SelectedItems = createContext<Set<unknown>>(null);
+const SelectedItems = createContext<Set<string>>(null);
 
 export function useSelectedDispatch() {
   return useContext(SelectedDispatch);
 }
 
-export function useSelectedItems() {
-  return useContext(SelectedItems);
+export function useSelectedItems<T = unknown>() {
+  return useContext<Set<T>>(SelectedItems);
```

Comment:
I can't think of a way to have stricter types here than `string` without some sort of casting or creating individual contexts per-caller. Maybe this might be okay?
```suggestion
export function useSelectedItems() {
  return useContext<Set<string>>(SelectedItems);
```